packages/nuxt/package.json

```
@@ -44,7 +44,7 @@
     "@nuxt/vite-builder": "3.0.0-rc.11",
     "@vue/reactivity": "^3.2.40",
     "@vue/shared": "^3.2.40",
-    "@vueuse/head": "^0.7.12",
+    "@vueuse/head": "^1.0.0-rc.5",
```

```suggestion
    "@vueuse/head": "1.0.0-rc.5",
```

Until it's stable I think it's better to pin the version

---

Very true

---

My only worry is if there are any bugs in `@vueuse/head` it will be difficult to fix them until the next Nuxt release. If you're happy to commit to no (intended) breaking changes between rc5 -> v1 then maybe we could consider `~` constraint?

wdyt @antfu?

---

I can't foresee the API that Nuxt is using will change, there may be some tinkering but nothing that should break.

If there is for whatever reason then I'll use a separate channel